.jira.d/config.yml

```diff
@@ -13,20 +13,72 @@ project: ADP
 custom-commands:
   - name: bugs
     help: List unresolved bug tickets
+    options:
+      - name: query
+        short: q
+    args:
+      - name: terms
+        required: false
     script: |-
-      {{jira}} list --template table --query "type = Bug and resolution = Unresolved and project=$JIRA_PROJECT and component = cardano-wallet ORDER BY priority DESC, updated DESC, rank ASC, created"
+      set -x
+      {{jira}} list --template table --query "type = Bug and resolution = Unresolved and project=$JIRA_PROJECT and component = cardano-wallet{{ with args.terms }} and text ~ \"{{ args.terms }}\"{{ end }}{{ with options.query }} and {{ options.query }}{{ end }} ORDER BY priority DESC, updated DESC, rank ASC, created"
 
   - name: mine
     help: Display unresolved issues assigned to me
     script: |-
       {{jira}} list --template table --query "resolution = Unresolved and assignee=currentuser() and project = $JIRA_PROJECT ORDER BY priority asc, created"
 
+  - name: sprint_base
+    hidden: true
+    args:
+      - name: template
+        required: true
+    script: |-
+      {{jira}} list --template sprint-{{ args.template}} --query "project=$JIRA_PROJECT AND sprint IN openSprints() AND type != epic ORDER BY rank ASC, created" --queryfields=issuetype,summary,status,assignee,priority,customfield_12502,description,customfield_10023,customfield_10021
+
+  - name: sprint-full
+    help: Markdown dump of issues in active sprint
+    script: |-
+      {{jira}} sprint_base full | pandoc -f jira -t markdown
+
   - name: sprint
```

Comment:
I thought the table-format was nicer than the list (but the new info is nice)

<img width="1116" alt="Skärmavbild 2021-04-14 kl  16 55 57" src="https://user-images.githubusercontent.com/304423/114731761-4a9ae680-9d42-11eb-8f1c-f3010ffef30d.png">

<img width="646" alt="Skärmavbild 2021-04-14 kl  17 10 55" src="https://user-images.githubusercontent.com/304423/114734303-630c0080-9d44-11eb-854c-9f38e50dee6e.png">

Reply:
OK - changed to table format.